moveit_core/robot_state/src/cartesian_interpolator.cpp

```diff
@@ -56,46 +56,45 @@ static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit_robot_state.cart
 
 CartesianInterpolator::Distance CartesianInterpolator::computeCartesianPath(
     RobotState* start_state, const JointModelGroup* group, std::vector<RobotStatePtr>& traj, const LinkModel* link,
-    const Eigen::Vector3d& direction, bool global_reference_frame, double distance, const MaxEEFStep& max_step,
+    const Eigen::Vector3d& translation, bool global_reference_frame, const MaxEEFStep& max_step,
```

Comment:
If you want to change it to translation, I'd suggest using the `Eigen::Translation3d` type. I like this simplification, though.

Reply:
Tried this, but changing the type to `Translation3d` make the code much more complicated, because you cannot easily pass `scalar * Eigen::Vector3d` anymore. It's debatable, but here we actually want to pass the translation direction, while `Translation3d` is a transform.

Reply:
In this case it would really be about semantics, but I see your point. It's not overly important here.

Reply:
I was talking about semantics too. We don't move towards a given (translation) _transformation_ (given w.r.t. local or global frame) but along some _direction vector_ :wink: